### PR TITLE
Added desktop file, test helper

### PR DESCRIPTION
**Pull request by lslezak**
- Just adding some basic files

### CODE REVIEW

File: test/test_helper.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# ------------------------------------------------------------------------------
+# Copyright (c) 2015 SUSE LLC, All Rights Reserved.
```
Comment (dgdavid) [inline test/test_helper.rb:2]: NP: Outdated header :) I would drop those `#--------` too.

File: src/desktop/org.opensuse.yast.Migration-SLE.desktop [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+X-SuSE-YaST-SortKey=
+X-SuSE-YaST-Keywords=sle,sles,enterprise,migration,registration
+
+Icon=yast-migration
+Exec=xdg-su -c "/sbin/yast2 migration-sle"
+
+Name=YaST Online Migration to SLES
+
+GenericName=Online SLES Migration
+Comment=Migrate to the SUSE Linux Enterprise Server product
```
Comment (dgdavid) [inline src/desktop/org.opensuse.yast.Migration-SLE.desktop:21]: Question, only to SLES or to SLE in general? I mean, SLE Desktop and other products.

Comment (dgdavid) [reply src/desktop/org.opensuse.yast.Migration-SLE.desktop:21]: Ok, answered in https://github.com/yast/yast-migration-sle/pull/1/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R14

> This YaST module allows online migration openSUSE Leap distribution to the
SUSE Linux Enterprise Server (SLES).

**Review by dgdavid — APPROVED** [review]

File: src/desktop/org.opensuse.yast.Migration-SLE.desktop (hunk 2 above)
Comment (lslezak) [reply src/desktop/org.opensuse.yast.Migration-SLE.desktop:21]: Currently only SLES, but in theory any SLE product, depends on the SCC.

**Pull request merged**